src/load_inputs/load_generators_variability.jl

```diff
@@ -10,7 +10,8 @@ function load_generators_variability!(setup::Dict, path::AbstractString, inputs:
     if setup["TimeDomainReduction"] == 1  && time_domain_reduced_files_exist(data_directory)
         my_dir = data_directory
 	else
-        my_dir = path
+        # If TDR is not used, then use the "system" directory specified in the setup
+        my_dir = joinpath(path, setup["SystemFolder"])
 	end
```

Comment:
Consider making these five lines to construct `my_dir` be a function (of `setup`, `data_directory`, and `path` I suppose) since they're repeated in `load_demand_data`, `load_fuels_data`, `load_generators_availability`, `load_vre_stor_availability`.